h2o-docs/src/product/data-science/gam.rst

```diff
@@ -579,3 +689,15 @@ References
 .. _ref2:
 
 2. T.J. Hastie, R.J. Tibshirani, Generalized Additive Models, Chapman and Hall, First Edition, 1990.
+
+.. _ref3:
+
+3. Lecture 7 Divided Difference Interpolation Polynomial by Professor R.Usha, Department of Mathematics, IITM, https://www.youtube.com/watch?v=4m5AKnseSyI .
+
+.. _ref4:
+
+4. Carl De Boor et. al., ON CALCULATING WITH B-SPLINES II. INTEGRATION, ResearchGate Article, January 1976.
+
+.. _ref5:
+
+5. J.O. Ramsay, “Monotone Regression Splines in Action”, Statistical Science, 1988, Vol. 3, No. 4, 425-461.
```

Comment:
Can you add a python/R example on how to use monotone spline like before for the other spline types?